src/workerd/api/trace.h

```diff
@@ -220,9 +220,9 @@ class TraceItem::EmailEventInfo final: public jsg::Object {
   uint32_t getRawSize();
 
   JSG_RESOURCE_TYPE(EmailEventInfo) {
-    JSG_READONLY_PROTOTYPE_PROPERTY(mailFrom, getMailFrom);
-    JSG_READONLY_PROTOTYPE_PROPERTY(rcptTo, getRcptTo);
-    JSG_READONLY_PROTOTYPE_PROPERTY(rawSize, getRawSize);
+    JSG_READONLY_INSTANCE_PROPERTY(mailFrom, getMailFrom);
+    JSG_READONLY_INSTANCE_PROPERTY(rcptTo, getRcptTo);
+    JSG_READONLY_INSTANCE_PROPERTY(rawSize, getRawSize);
```

Comment:
Is this because the structure is JSON stringified? If so, a better long term solution here is to implement a toJSON method. This change is good for now but let me explore the toJSON approach separately

Reply:
That is correct. This Event object will be JSON.stringified() by the Tracer in edgeworker.

Reply:
ok yeah, I'll work up a generic toJSON mechanism that will make this easier. In general the reason we want to avoid `JSG_*_INSTANCE_PROPERTY` is that it forces the objects to stay alive longer than they otherwise need to be rather than being gc'd earlier.